src/experimental-layers/src/text-layer/text-layer.js

```diff
@@ -63,7 +63,7 @@ export default class TextLayer extends CompositeLayer {
   }
 
   updateState({props, oldProps, changeFlags}) {
-    if (changeFlags.dataChanged || changeFlags.updateTriggers.getText) {
+    if (changeFlags.dataChanged || props.getText) {
```

Comment:
@ibgreen not sure if it is the right fix, cause I don't know the intention of this check.

Reply:
It should be `changeFlags.updateTriggersChanged.getText`.

Good catch!